docs/scripts/util/checkLink.ts

```diff
@@ -0,0 +1,87 @@
+import https from 'https';
+import http from 'http';
+import { IGNORED_LINKS } from '../../src/data/ignoredLinks';
+
+const requestedUrl: Set<string> = new Set();
+export async function checkLink(
+  {
+    href,
+    tagName,
+    tagText,
+    pageIdx,
+    pageUrl,
+  }: {
+    href: string;
+    tagName: string;
+    tagText: string;
+    pageIdx: string;
+    pageUrl: string;
+  },
+  linkIdx: number
+) {
+  return new Promise(async (res, rej) => {
+    if (!href) {
+      console.log(
+        `⚠️[WARNING...] page #${pageIdx} link #${linkIdx} "${tagName}" tag "${tagText}" doesn't have a href.`
```

Comment:
Maybe we should throw an error and fix all of the empty hrefs in this PR?

Reply:
IMO this can be another PR